docs/bigtable-data-api.rst

```diff
@@ -0,0 +1,328 @@
+Data API
+========
+
+After creating a :class:`Table <gcloud.bigtable.table.Table>` and some
+column families, you are ready to store and retrieve data.
+
+Cells vs. Columns vs. Column Families
++++++++++++++++++++++++++++++++++++++
+
+* As we saw before, a table can have many column families.
```

Comment:
It's possible users came to this page without first reading the table page. Better to link there and say "As explained in the [table]() overview, tables can have many column families"